packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -367,25 +405,174 @@ const TitleIcon = ({
 }: TitleIconProps) => {
   const StatusIcon = !!status && StatusIconMap[status];
 
+  let icon = null;
+
   if (isLoading) {
-    return (
-      <ProgressRadial
-        className='iui-tile-status-icon'
-        aria-hidden
-        indeterminate
-      />
+    icon = (
+      <ProgressRadial style={{ height: '100%' }} aria-hidden indeterminate />
```

Comment:
why inline style `height: 100%`?

Reply:
What should we do about this now?

Reply:
For now, you'll need to adjust the CSS to fix it. I think changing `svg` to `svg:not(.iui-radial)` might do it?

I'm working on changes to `<ProgressRadial>` which will avoid the need for such hacks.